homeassistant/components/roomba/config_flow.py

```diff
@@ -180,7 +191,7 @@ async def async_step_manual(self, user_input=None):
             return self.async_abort(reason="already_configured")
 
         self.host = user_input[CONF_HOST]
-        self.blid = user_input[CONF_BLID]
+        self.blid = user_input[CONF_BLID].upper()
```

Comment:
Since the BILD is now uppercase and used as a unique ID, should existing entries be migrated?

Reply:
They should all be uppercase since we only started getting lowercase data from dhcp in this version.